AsyncDisplayKit/Layout/ASRelativeLayoutSpec.mm

```diff
@@ -52,11 +52,15 @@ - (void)setSizingOption:(ASRelativeLayoutSpecSizingOption)sizingOption
   _sizingOption = sizingOption;
 }
 
-- (ASLayout *)measureWithSizeRange:(ASSizeRange)constrainedSize
+- (ASLayout *)calculateLayoutThatFits:(ASSizeRange)constrainedSize
 {
+  // If we have a finite size in any direction, pass this so that the child can
+  // resolve percentages against it. Otherwise pass ASLayoutableParentDimensionUndefined
+  // as the size will depend on the content
+  // TODO: layout: isValidForLayout() call should not be necessary if INFINITY is used
```

Comment:
The goal is to remove all TODOs in time we merge the PR. One of the open tasks in the PR description is to remove this TODOs.